hail/python/hail/methods/impex.py

```diff
@@ -159,7 +159,6 @@ def export_gen(dataset, output, precision=4, gp=None, id1=None, id2=None,
     dataset = dataset._select_all(col_exprs=sample_exprs,
                                   col_key=[],
                                   row_exprs=gen_exprs,
-                                  row_key=['locus', 'alleles'],
```

Comment:
Why can you delete these?

Reply:
`export_gen` assumes locus-alleles row key, so the `_select_all` isn't changing the row key. I'm pretty sure it would have been equivalent before to leave it out, but now leaving it out avoids generating unnecessary `MatrixKeyRowsBy` IR nodes.